hdfs_namenode/datadog_checks/hdfs_namenode/data/conf.yaml.example

```diff
@@ -210,13 +325,16 @@ instances:
     #
     # persist_connections: false
 
-## Log Section (Available for Agent >=6.0)
+## Log Section
 ##
-## type - mandatory - Type of log input source (tcp / udp / file / windows_event)
-## port / path / channel_path - mandatory - Set port if type is tcp or udp. Set path if type is file. Set channel_path if type is windows_event
-## service - mandatory - Name of the service that generated the log
-## source  - mandatory - Attribute that defines which Integration sent the logs
-## tags: - optional - Add tags to the collected logs
+## type - required - Type of log input source (tcp / udp / file / windows_event)
+## port / path / channel_path - required - Set port if type is tcp or udp.
+##                                         Set path if type is file.
+##                                         Set channel_path if type is windows_event.
+## source  - required - Attribute that defines which Integration sent the logs
+## service - required - The name of the service that generates the log.
```

Comment:
remove this one, we're favoring the global service config

Reply:
Shouldn't it be removed from the template then?